src/main.rs

```diff
@@ -4,10 +4,12 @@
 #![feature(never_type)]
 #![feature(custom_test_frameworks)]
 #![feature(const_raw_ptr_to_usize_cast)]
+#![feature(const_in_array_repeat_expressions)]
```

Comment:
What is this needed for?

Reply:
For ``static mut PageUsage: [UsageFlags; PAGE_NUMBER] = [UsageFlags::Free; PAGE_NUMBER];`` in src/allocator/mod.rs:7